OpenRA.Mods.Common/Traits/Player/StrategicVictoryConditions.cs

```diff
@@ -23,7 +23,7 @@ public class StrategicPoint { }
 	public class StrategicVictoryConditionsInfo : ITraitInfo, Requires<MissionObjectivesInfo>
 	{
 		[Desc("Amount of time (in game ticks) that the player has to hold all the strategic points.", "Defaults to 5 minutes.")]
```

Comment:
Come to think of it, this is wrong since we've got gamespeeds working. "Defaults to 7500 ticks (5 minutes in default speed)."?